src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class FigureSupplier {
+    private static final double NUMBER_BOUND = 50.0;
+    private final SecureRandom random = new SecureRandom();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Why final? in this case, nothing critical, but in the future it is better not to do this. Best practice to create constructor